api/v1alpha1/bundle_types.go

```diff
@@ -50,6 +50,7 @@ const (
 
 // BundleSpec defines the desired state of Bundle
 type BundleSpec struct {
+	//+kubebuilder:validation:Enum:={"core.rukpak.io/plain","core.rukpak.io/registry",}
```

Comment:
I think this makes sense, but it directly conflicts with rukpak being an open framework of APIs that users can plug into with their own provisioner. For example, it's at odds with #425 at a minimum, also #463. I'm not sure we want to be this opinionated regarding the rukpak APIs, at least in the upstream project.

Reply:
I see.  We probably wait until we define the way to register provisioner before we implement this validation function.